frontend/javascripts/test/model/binary/pullqueue.spec.js

```diff
@@ -107,7 +107,9 @@ test.serial("Successful pulling: should receive the correct data", t => {
 
 function prepare() {
   WkstoreAdapterMock.requestWithFallback = sinon.stub();
-  WkstoreAdapterMock.requestWithFallback.onFirstCall().returns(Promise.reject());
+  WkstoreAdapterMock.requestWithFallback
+    .onFirstCall()
+    .returns(Promise.reject(new Error("Expected promise rejection in tests. Can be ignored.")));
```

Comment:
I spent ages hunting down this promise rejection which let the test fail (for a good reason!, I found a bug that way), but because no rejection reason was given, there was no stack trace or anything, so I didn't find it. I even updated ava to the newest version in the process because I thought I would get more information :see_no_evil: In the end, I found it through the debugger and we use the newest ava version now, so there's that :)